collector/project.go

```diff
@@ -106,6 +110,35 @@ func (p Project) Collect(c *CollectorOpts) interface{} {
 			p.Pod.Update(totalPo)
 			poUtils = append(poUtils, float64(totalPo))
 		}
+
+		// Apps
+		defaultLibraryName := ""
+
+		catalogCollection, err := c.Client.Catalog.List(&opts)
+		if err != nil {
+			log.Error("Error listing catalogs")
+			continue
+		}
+
+		for _, catalog := range catalogCollection.Data {
+			if catalog.URL == "https://git.rancher.io/charts" {
+				defaultLibraryName = catalog.Name
+			}
+		}
+
+		appsCollection := GetAppsCollection(c, project.Links["apps"])
+		if appsCollection != nil {
+			for _, app := range appsCollection.Data {
+				if strings.Contains(app.ExternalID, fmt.Sprintf("catalog=%s&", defaultLibraryName)) {
```

Comment:
If the default catalog isn't found this ends up finding ALL catalogs because defaultLibraryName="".  Assume it's called library and bail early if the URL isn't right.

Reply:
I fixed this in the new version, also because I'm assuming we only want to parse if `defaultLibraryName` is the rancher one: https://github.com/rancher/telemetry/pull/33/files#diff-b071aa59b9ae4ff35355cee91b5e005bR128